include/vcpkg/base/message-data.inc.h

```diff
@@ -1477,6 +1477,11 @@ DECLARE_MESSAGE(HelpBinaryCachingAzBlob,
                 "**Experimental: will change or be removed without warning**\n"
                 "Adds an Azure Blob Storage source. Uses Shared Access Signature validation. <url> should include "
                 "the container path. <sas> must be be prefixed with a \"?\".")
+DECLARE_MESSAGE(HelpBinaryCachingAzUpkg,
+                (),
+                "Printed as the 'definition' for 'x-upkg,<organization>,<project>,<feed>[,<rw>]'.",
```

Comment:
Hmmmm given that we are going to tell important customers to use this meaning we are unlikely to mess with it again perhaps it should be born not experimental? (Should we not have the `x-`?)

Reply:
```suggestion
                "Printed as the 'definition' for 'upkg,<organization>,<project>,<feed>[,<rw>]'.",
```

The product code dropped the x